lib/ZwaveClient.js

```diff
@@ -374,12 +374,13 @@ function onNodeDead (zwaveNode, oldStatus) {
 
 function onNodeValueAdded (zwaveNode, args) {
   logger.info(
-    `Node ${zwaveNode.id}: value added: ${getValueID(args)} =>`,
-    args.newValue
+    `Node ${zwaveNode.id}: adding value ${getValueID(args)} => ${args.newValue}`
   )
-
   // handle node values added 'on fly'
-  if (zwaveNode.ready) {
+  if (zwaveNode.ready || args.newValue) {
+    if (args.newValue) {
+      args.value = args.newValue
```

Comment:
Also I don't understand why and how this could fix the problem, I use the zwave-js api getValue to get the actual value later

Reply:
when you refresh (refresh info) a node, this never updates the values of the node, and reverts them to undefined. 
Because node is not ready, this never been executed. Refresh Info send newValue instead of value.
So newValue is parsed later? (just did not find it)

Reply:
> So newValue is parsed later? (just did not find it)

I never use that, I set the value in the `parseValue` function here: https://github.com/zwave-js/zwavejs2mqtt/blob/master/lib/ZwaveClient.js#L712